plugins/node/opentelemetry-instrumentation-mysql2/src/types.ts

```diff
@@ -15,5 +15,18 @@
  */
 
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import type { Span } from '@opentelemetry/api';
 
-export type MySQL2InstrumentationConfig = InstrumentationConfig;
+export interface MySQL2InstrumentationExecutionResponseHook {
+  (span: Span, queryResults: any): void;
```

Comment:
Please make the second parameter an `info` object so adding more data in the future will not create long function signatures or breaking changes.
You can see examples in other instrumentations

Reply:
done